test/unit/registry-routes-helpers-get-component.js

```diff
@@ -96,4 +96,72 @@ describe('registry : routes : helpers : get-component', () => {
       expect(fireStub.args[0][1].status).to.equal(500);
     });
   });
+
+  describe("when oc-client request an unrendered component and it doesn't support the correct template", () => {
```

Comment:
very minor but I would still use single quotes and either escape the apostrophe w/ a '\' or even consider to use '`' instead. 

also the describe should be `when oc-client requests ...`